payments-ui-core/src/main/java/com/stripe/android/ui/core/PaymentsTheme.kt

```diff
@@ -78,17 +78,30 @@ data class PaymentsTypography(
 )
 
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
-data class PrimaryButtonModifier(
-    val primaryLight: Color,
-    val onPrimaryLight: Color,
-    val primaryDark: Color,
-    val onPrimaryDark: Color,
+data class PrimaryButtonTheme(
```

Comment:
In my mind Theme kind of evokes something that would apply across an entire app, and Style makes me think more of an individual element.